yarn-project/canary/Dockerfile

```diff
@@ -1,16 +1,29 @@
-FROM node:18-alpine
+FROM 278380418400.dkr.ecr.us-east-2.amazonaws.com/yarn-project-base AS builder
+
 RUN apk update && apk add --no-cache udev ttf-freefont chromium curl jq bash
 ENV CHROME_BIN="/usr/bin/chromium-browser" PUPPETEER_SKIP_CHROMIUM_DOWNLOAD="true"
 
-ARG COMMIT_TAG=""
+ARG COMMIT_TAG="0.7.5"
```

Comment:
@spypsy Should this have a default value? Or should it have an invalid default value so we catch the case where COMMIT_TAG isn't being provided?

Reply:
You're right, it's better to hardcode an invalid tag so it fails with a logged error.
Fixed here: https://github.com/AztecProtocol/aztec-packages/pull/2371/files